Tools/AP_Periph/can.cpp

```diff
@@ -1503,6 +1563,11 @@ void AP_Periph_FW::can_start()
 #endif
         instances[i].iface = can_iface_periph[i];
         instances[i].index = i;
+#ifdef HAL_PERIPH_CAN_MIRROR
```

Comment:
shouldn't this be inside the below HAL_NUM_CAN_IFACES area? Is there ever a scenario where you'd mirror with only 1 CAN interface?

Reply:
There isn't, but that's sorta an HWDEF error. I can add #error that requires HAL_NUM_CAN_IFACES if you want the mirror, but I'd prefer to not have to litter all the MIRROR checks with an additional check.

Reply:
Done as proposed.